src/components/IstioConfigPreview/IstioConfigPreview.tsx

```diff
@@ -20,62 +27,67 @@ import { EditResources } from './EditResources';
 import { cloneDeep } from 'lodash';
 import _ from 'lodash';
 
-type PolicyItem = AuthorizationPolicy | Sidecar;
+export type PolicyItem =
+  | AuthorizationPolicy
+  | Sidecar
+  | DestinationRule
+  | PeerAuthentication
+  | Gateway
+  | VirtualService;
+
+export interface ConfigPreviewObject {
```

Comment:
ConfigPreviewObject -> "ConfigPreviewItem"?
Just to keep the terminology used?

(it's just naming, but that helped me to follow better the flow of the changes)